server/handlers.go

```diff
@@ -40,7 +40,7 @@ func getUserFirstName(c *gin.Context) {
 	id := claims["id"]
 	var user models.User
 	db.Where("id = ?", id).First(&user)
-	c.JSON(200, user.FName)
+	c.String(200, user.FName)
```

Comment:
Revert it back to JSON.
If there is some problem in testing then, typecast the JSON to string during test assertion

Reply:
Updated in the commit 46c5e524ee9090fa246d58ea5922440dd6f7147c